SonarQube.TeamBuild.PreProcessor/Roslyn/RoslynAnalyzerProvider.cs

```diff
@@ -48,28 +48,28 @@ public RoslynAnalyzerProvider(IAnalyzerInstaller analyzerInstaller, ILogger logg
             this.logger = logger;
         }
 
-        public AnalyzerSettings SetupAnalyzers(ISonarQubeServer sqServer, TeamBuildSettings teamBuildSettings, string sqProjectKey, string sqProjectBranch)
+        public AnalyzerSettings SetupAnalyzers(ISonarQubeServer server, TeamBuildSettings settings, string projectKey, string projectBranch)
```

Comment:
I renamed the parameters to match the names used in the interface.
This then causes another issue that the parameter names are the same as the field names so I renamed the field names.